src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/helper/builders/OffenderBookingBuilder.kt

```diff
@@ -20,6 +21,8 @@ class OffenderBookingBuilder(
   var sentences: List<SentenceBuilder> = emptyList(),
   var adjudications: List<Pair<AdjudicationIncident, AdjudicationPartyBuilder>> = emptyList(),
   var keyDateAdjustments: List<KeyDateAdjustmentBuilder> = emptyList(),
+  var offenderProgramProfiles: List<OffenderProgramProfileBuilder> = emptyList(),
+  val repository: Repository? = null,
```

Comment:
Now need to pass a repository in for the course allocation builder. The long term goal is to split up Repository and for each builder to be autowired the repos it needs - but until then we need to pass Repository around.